README.md

```diff
@@ -26,6 +28,9 @@ beats_client_beats_packages:
   - filebeat
   - metricbeat
 
+beats_client_version: 5.5.0
+beats_client_major_ver: 5.x
```

Comment:
Would perform more consistent variable names here, e.g. 

```yaml
beats_client_full_version: 5.5.0
beats_client_major_version: 5.x
```

or 

```yaml
beats_client_version: 5.5.0
# The apt repo URL pegs minor versions (e.g. 5.x) to avoid unexpected upgrades
beats_client_major_version_abbreviated: 5.x
```